clients/banking/src/components/CheckoutPaymentPage.tsx

```diff
@@ -0,0 +1,3 @@
+export const CheckoutPaymentPage = () => {
+  return <>Test</>;
```

Comment:
Text must be in `<Text>` component or react-native-web will not be happy about it (console.error)